src/globus_sdk/services/flows/client.py

```diff
@@ -448,6 +448,47 @@ def delete_flow(
 
         return self.delete(f"/flows/{flow_id}", query_params=query_params)
 
+    def get_run(
+        self, run_id: UUIDLike, *, include_flow_description: bool = False
```

Comment:
Do we want a `query_params` argument for passing additional arbitrary query params here?

Reply:
If so `get_run_logs` should probably have one as well

Reply:
Ah, good catch! Yes, I like for us to have that on any GET wrappers.

The practical reality is that GET is only extensible in very limited ways, primarily query params. So having `query_params={"foo": "bar"}` support lets older SDK versions use newer features.

Reply:
Sure, I can add a query_params to this PR & [get_run_logs](https://github.com/globus/globus-sdk-python/pull/722)

New Revs

Reply:
Sorry about the pylint failure, but the update looks good!
I'll wait for CI to pass, but let's close this thread. :smile: